lib/testing/Makefile

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
 
-USER_TARGETS :=
+USER_TARGETS := test-tool
```

Comment:
With this, 'make install' will install the binary into `/usr/sbin`. I think common.mk will need a bit of adjustment to have it go into `/usr/share/xdp-tools` along with the tests instead...